Week3/db-week-3.js

```diff
@@ -0,0 +1,216 @@
+'use strict';
+const util = require('util');
+const fs = require('fs');
+const mysql = require('mysql');
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const port = 3020;
+
+const todoDatabase = {
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'todo',
+  multipleStatements: true,
+};
+
+app.set('view engine', 'pug');
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(__dirname + '/public'));
+
+const CREATE_TABLE_TODOS = `CREATE TABLE IF NOT EXISTS todos ( 
+  todoid VARCHAR(30) PRIMARY KEY,
+  reminder text 
+  );`;
+
+const CREATE_TABLE_ITEMS = `CREATE TABLE IF NOT EXISTS items (
+    item VARCHAR(30),
+    isdone enum('Yes', 'No'),
+    belongs_to VARCHAR(30),
+    
+    FOREIGN KEY (belongs_to)
+    REFERENCES todos(todoid)
+    ON DELETE CASCADE
+    );`;
+
+app.get('/', async function(req, res) {
+  try {
+    const connection = mysql.createConnection(todoDatabase);
+    const execQuery = util.promisify(connection.query.bind(connection));
+    await execQuery(CREATE_TABLE_TODOS);
+    await execQuery(CREATE_TABLE_ITEMS);
+    res.sendFile(__dirname + '/pages/index.html');
+    connection.end();
+  } catch (err) {
+    if (err) {
+      throw err;
+    }
+  }
+});
+
+app.get('/download', function(req, res) {
+  const file = `${__dirname}/instructions/Readme.md`;
+  res.download(file);
+});
+
+app.post('/submit', async function(req, res) {

```

Comment:
It's usually a good practice to have self descriptive names, for instance:

submitReview, submitForm, submitHomework, etc instead of just submit.

The idea is that the person reading the code has an idea of what your function does without the need to read the body of the function itself. The same with variable names :)